lib/net/http/header.rb

```diff
@@ -610,15 +610,40 @@ def set_range(r, e = nil)
 
   alias range= set_range
 
-  # Returns an Integer object which represents the HTTP Content-Length:
-  # header field, or +nil+ if that field was not provided.
+  # Returns the value of field <tt>'Content-Length'</tt> as an integer,
+  # or +nil+ if there is no such field;
+  # see {Content-Length request header}[https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#content-length-request-header]:
+  #
+  #   res = Net::HTTP.start(hostname) do |http|
+  #     http.get('/nosuch/1')
+  #   end # => #<Net::HTTPNotFound 404 Not Found readbody=true>
```

Comment:
Wouldn't it be easier to use `Net::HTTP.get` here to make it one line?

Reply:
I wanted to emphasize the use of start, on the theory that many users will have more than one request to make and should be aware of it.  There are a number of these (16?) in this file.  If one changes, should all?

Reply:
So, change all?